node/src/components/consensus/era_supervisor.rs

```diff
@@ -151,8 +151,8 @@ where
         let (root, config) = config.into_parts();
         let (secret_signing_key, public_signing_key) = config.load_keys(root)?;
         info!(our_id = %public_signing_key, "EraSupervisor pubkey",);
-        let metrics = ConsensusMetrics::new(registry)
-            .expect("failure to setup and register ConsensusMetrics");
+        let metrics =
+            Metrics::new(registry).expect("failure to setup and register ConsensusMetrics");
```

Comment:
Technically this message should now be `"failed to setup and register consensus metrics"`, since it does not denote a type anymore? Nittiest of nitpicks though.

Reply:
Good spot.  I'll sneak that into my next PR.